net-news/liferea/liferea-1.12.6b.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit gnome2 python-single-r1
+
+DESCRIPTION="News Aggregator for RDF/RSS/CDF/Atom/Echo feeds"
+HOMEPAGE="https://lzone.de/liferea/"
+SRC_URI="https://github.com/lwindolf/${PN}/releases/download/v1.12.6/${P}.tar.bz2"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~x86"
+IUSE="gnome-keyring mediaplayer networkmanager notification tray"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="${PYTHON_DEPS}
```

Comment:
This word is not used by current stable portage as I see. Why we should avoid it?

Reply:
Because future EAPIs might start to use it (any [*]DEPEND), and __then__ it becomes a hassle updating all ebuilds. You can do preventive work to avoid that, now ;)

Reply:
Until it is not documented and usage of ```[A-Z]DEPEND``` variables is not explicitly prohibited I do not see any reason to change it.
```
> find /var/db/repos/gentoo/ -name '*.ebuild' | xargs grep CDEPEND | wc -l
3748
```

Reply:
It may not be outright banned but it is already suggested: https://wiki.gentoo.org/wiki/Future_EAPI/New_dependency_types